monkey/infection_monkey/utils/commands.py

```diff
@@ -0,0 +1,15 @@
+import shlex
+
+
+def get_monkey_cmd_lines_windows(monkey_cmdline_windows, destination_path, monkey_options):
+    monkey_cmdline = monkey_cmdline_windows % {"monkey_path": destination_path} + monkey_options
+
+    return monkey_cmdline, shlex.split(monkey_cmdline, posix=False)
+
+
+def get_monkey_cmd_lines_linux(monkey_cmdline_linux, destination_path, monkey_options):
+    monkey_cmdline = (
+        monkey_cmdline_linux % {"monkey_filename": destination_path.split("/")[-1]} + monkey_options
+    )
+
+    return monkey_cmdline, shlex.split(monkey_cmdline, posix=False)
```

Comment:
The point of these TODOs was to get rid of the string templating altogether. `subprocess.Popen()` accepts a list of arguments. There's no need for us to build up a string of arguments and then use `shlex.split()` to break it into a list. We can just build a list to begin with.

Modifications will need to be made to `build_monkey_commandline_explicitly()` and subsequently `build_monkey_commandline()`. We can avoid modifying all of the exploiters for right now by having `build_monkey_commandline()` join the output of `build_monkey_commandline_explicitly()` into to a string. We should leave a TODO there, so we come back and rework the exploiters later down the road.

We'll also need unit tests for everything we touch :)